indico/modules/events/editing/client/js/management/editable_type/EditableList.jsx

```diff
@@ -63,17 +103,52 @@ export default function EditableList() {
     setSortedList(_sortList({sortBy, sortDirection}));
   };
 
+  const renderCode = code => code || Translate.string('n/a');
+  const renderEditor = editable => {
+    return editable && editable.editor ? (
+      <div>
+        <Icon name="user outline" />
+        {editable.editor.fullName}
+      </div>
+    ) : (
+      <div />
+    );
+  };
+  // eslint-disable-next-line no-shadow
+  const renderTitle = (title, index) => {
+    return sortedList[index].editable ? (
+      <a href={sortedList[index].editable.timelineURL}>{title}</a>
+    ) : (
+      <div>{title}</div>
+    );
+  };
+  const renderStatus = editable => {
+    return <StateIndicator state={editable ? editable.state : 'not_submitted'} circular label />;
+  };
+  const renderFuncs = {
+    title: renderTitle,
+    code: renderCode,
+    status: renderStatus,
+    editor: renderEditor,
+  };
+
   // eslint-disable-next-line react/prop-types
   const renderCell = ({dataKey, rowIndex}) => {
+    let data = sortedList[rowIndex][dataKey];
+    if (dataKey === 'status' || dataKey === 'editor') {
+      data = sortedList[rowIndex]['editable'];
+    }
+    const fn = renderFuncs[dataKey] || (x => x);
     return (
       <TooltipIfTruncated>
-        <div index={rowIndex} styleName="rowcolumn-tooltip" role="gridcell">
-          {dataKey === 'assigned' && <Icon name="user outline" />}
-          {dataKey === 'status' ? (
-            <StateIndicator index={rowIndex} state={sortedList[rowIndex][dataKey]} circular label />
-          ) : (
-            sortedList[rowIndex][dataKey]
-          )}
+        <div
+          id={dataKey}
```

Comment:
is this ID needed for anything? it's adding tons of duplicate IDs..

Reply:
I don't think so. Removing it